### DISCUSSION TITLE
Encoding option for CLI convert

### DISCUSSION THREAD

**Discussion by joaompfe** [explosion/spaCy#8437] [New Features & Project Ideas]
Hello,

Why there's no encoding option available for CLI `convert`? Any special reason?

[spacy/cli/convert.py](https://github.com/explosion/spaCy/blob/83fd04dee503f98259c4bd87b401f14c4edf231c/spacy/cli/convert.py) has UTF-8 encoding hardcoded when opening the file.

Thanks

**Comment by joaompfe** [comment]
I have added the optional argument and changed the CLI documentation [here](https://github.com/joaompfe/spaCy/commit/8ac904723071af8f5de1c9558153d540bd0456ff). 

Tested on my machine the option `--encoding="ISO-8859-1"` with success. But did not implement any test.

Should I open a pull request?

**Reply by polm** [reply]
Thanks for the offer but I don't think we need this. 

spaCy only deals with utf-8 internally, just like Python 3, and I honestly can't remember the last time someone even mentioned it. While there are still reasons to work with other encodings these days, they're limited, and converting your data on the command line before giving it to spaCy isn't complicated.
